hw_3/templates/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Registration Form</title>
+</head>
+<body>
+    <h1>Registration Form</h1>
+
+    <form method="post">
+        {{ form.hidden_tag() }}
+
+        <label for="first_name">First Name:</label>
+        {{ form.first_name(size=20) }}<br>
+
+        <label for="last_name">Last Name:</label>
+        {{ form.last_name(size=20) }}<br>
+
+        <label for="email">Email:</label>
+        {{ form.email(size=30) }}<br>
+
+        <label for="password">Password:</label>
+        {{ form.password(size=20) }}<br>
+
+        <label for="confirm_password">Confirm Password:</label>
+        {{ form.confirm_password(size=20) }}<br>
+
+        <input type="submit" value="Зарегистрироваться">
```

Comment:
Зачем создавать кнопку, если она уже содержится внутри формы?
```suggestion
        {{ form.submit() }}
```